Testing/.editorconfig

```diff
@@ -19,6 +19,10 @@ charset = utf-8-bom
 # Expression-bodied members
 csharp_style_expression_bodied_methods = true:none
 
+# Ignore warnings and suggestions for outside using statement placement
+dotnet_style_qualification_for_field = false:suggestion
+
+
```

Comment:
Remove this rule from this config.  This will automatically inherit the newly added rule from the solution-wide config.  This is why you do not need to add it in both.

This info is all in the issue description.
![image](https://github.com/KinsonDigital/CASL/assets/105235056/bffc603b-ae63-4f6d-a4e7-7dc593d48b8a)



```suggestion
```